crates/holochain/src/conductor/p2p_agent_store.rs

```diff
@@ -83,7 +83,7 @@ pub fn list_all_agent_info_signed_near_basis(
     environ: EnvWrite,
     _kitsune_space: Arc<kitsune_p2p::KitsuneSpace>,
     basis_loc: u32,
-    limit: u32,
+    limit: usize,
```

Comment:
Why switch to a type that is a different size on different systems? I personally feel like the bit-size consistency is worth the occasional `as usize` when needed.

Reply:
Fair enough, I'll change it back